package.json

```diff
@@ -70,6 +70,7 @@
     "eslint-plugin-jsx-a11y": "^6.1.2",
     "eslint-plugin-react": "^7.11.1",
     "husky": "^1.1.2",
+    "isobject": "^3.0.1",
```

Comment:
any reason you chose this package specifically? can we not use `typeof`?

Reply:
It's a 1 liner package that i can just move into a util. But the package also checks that it's not an array